.storybook/__mocks__/webextension-polyfill.js

```diff
@@ -1,8 +0,0 @@
-module.exports = {
```

Comment:
Another thing we could do is move this file from `__mocks__` to just `mocks`. The version of this in `ui` seems to be there for Jest tests, but as this is inside of the Storybook config directory, this file doesn't seem to have anything to do with tests, it's just for Storybook. In fact Jest shouldn't even be trying to read this file, but it sees `__mocks__` directories as special: https://jestjs.io/docs/manual-mocks. So in order to keep Storybook configuration and the rest of the app separate I'm wondering if we should keep this file but just rename where it is.

Reply:
While that would be a reasonable solution, I do see the benefit in keeping both implementations together. Both the UI Jest mock and the Storybook mock are intended to represent a _complete_ mock of this API for the UI layer.

Using the same file for both would make sense to me. If we kept them separate, we'd need to sync them up regularly anyway.